packages/vite-node/src/client.ts

```diff
@@ -205,6 +205,12 @@ export class ViteNodeRunner {
     if (transformed[0] === '#')
       transformed = transformed.replace(/^\#\!.*/, s => ' '.repeat(s.length))
 
+    // #855 make `default` property configurable
```

Comment:
This will break source maps. This transformations should be done in either plugin or by Vite itself.